lms/djangoapps/instructor_task/tasks_helper/certs.py

```diff
@@ -39,11 +39,11 @@ def generate_students_certificates(
     students_to_generate_certs_for = CourseEnrollment.objects.users_enrolled_in(course_id)
 
     student_set = task_input.get('student_set')
-    if student_set == 'all_whitelisted':
```

Comment:
I assume these aren't used in any MFE?

Reply:
Nope! Only uses were in edx-platform. I also verified this morning that the new learning MFE redirects to the Instructor Dashboard of the legacy courseware.